packages/@coorpacademy-components/src/atom/icon/index.js

```diff
@@ -62,8 +62,8 @@ const Icon = React.memo(function Icon({
     <div className={style.iconWrapper} style={iconWrapperStyle}>
       <FontAwesomeIcon
         icon={`fa-${iconName}`}
-        color={effectiveIconColor}
-        font-size={effectiveSize.faSize}
+        color={iconColor ?? effectiveIconColor}
```

Comment:
you don't need to do this, the line above is already including it 
`const effectiveIconColor =
    iconColor || backgroundColor ? getForegroundColor(backgroundColor) : DEFAULT_ICON_COLOR;`

Reply:
without the modify, the outcome color is not as expected for both normal and hover state

<img width="279" alt="image" src="https://github.com/CoorpAcademy/components/assets/160047673/93a82338-d0ec-40b1-812c-70ced767fea8">


<img width="279" alt="image" src="https://github.com/CoorpAcademy/components/assets/160047673/201de2ea-3439-4703-a0e8-51271fd9bbfc">

Reply:
@HuyIT9898 
this should resolve your problem, yeah just parentheses :p 
`
  const effectiveIconColor =
    iconColor || (backgroundColor ? getForegroundColor(backgroundColor) : DEFAULT_ICON_COLOR);
`